package.json

```diff
@@ -1832,7 +1832,7 @@
             "panel": [
                 {
                     "id": "jupyter-variables",
-                    "title": "Jupyter",
+                    "title": "",
```

Comment:
Are there any side effects of this? Like the tooltip for the notebook icon when the variable explorer panel is docked in the sidebar might be blank if you don't specify a title?

Reply:
As I recall with the old UI there were cases where the view container had loaded, but not the view itself. Might mean that for a brief period there is some type of empty, nameless view in the UI. At the same time, I don't know of any specific issues with this approach, so doesn't seem like the worst thing in the world to give it a try and see if anything shakes out.